### ISSUE TITLE
Feature suggestion: Add ons for nextcloud

### ISSUE THREAD

**Issue by storm1er** [photoprism/photoprism#1271]
Hi,

It would be awesome to have an integration directly into NextCloud.

A raw suggestion would be to have a new page with embedded PhotoPrism in it.  
Going further we could replace the original photo page.

This would also simplify sync and other stuff for people willing to use both :)   
Just to know, is this idea a good one, do people could like it?

**Comment by RealCommerzpunk** [comment]
You can do that today, using the external sites App / AddOn.
https://apps.nextcloud.com/apps/external

It can look like this (my setup)

https://i.imgur.com/hUC7HiC.png

**Comment by storm1er** [comment]
@RealCommerzpunk I agree this is a possibility, but it's still an external website that needs a server, installation, and more. 
Also, this way you lose per user feature, you have one more login to do and remember, etc.

Which is not an easy way for _non-tech_ people.

The idea here is to get what you have, but easier as it would not be on its own server instance/vm/container.

**Comment by ameralzarouni** [comment]
@RealCommerzpunk where did you get the icon from?
